packages/web-runtime/src/composables/head/useHead.ts

```diff
@@ -7,13 +7,21 @@ export const useHead = ({ store }: { store: Store<any> }) => {
   _useHead(
     computed(() => {
       const favicon = store.getters['configuration']?.currentTheme?.logo?.favicon
+      const themeColor = store.getters['configuration']?.currentTheme?.themeColor
       return {
         meta: [
           {
             name: 'generator',
             content: [getWebVersion(), getBackendVersion({ store })].filter(Boolean).join(', ')
+          },
+          themeColor && {
+            name: 'theme-color',
+            content: '#1b223d'
```

Comment:
Where do you chose this color from? 🤔

Reply:
i have taken from the website of the owncloud it is the color of nav bar. Is there is any problem I can change the theme color just give me the theme color I will add it

Reply:
@Shu12388y please see my other comment in the PR regarding the CSS variable